core-processor/src/ext.rs

```diff
@@ -491,7 +493,7 @@ impl EnvExt for Ext {
         let (common_charge, allowance_charge) = charge_gas_token!(self, costs);
 
         let res: Result<(), ProcessorError> = match (common_charge, allowance_charge) {
-            (NotEnough, _) => Err(ExtError::GasLimitExceeded.into()),
+            (NotEnough, _) => Err(ExecutionError::GasLimitExceeded.into()),
             (Enough, NotEnough) => Err(TerminationReason::GasAllowanceExceeded.into()),
             (Enough, Enough) => Ok(()),
         };
```

Comment:
.map_err(Into::into)?

Reply:
I think it's uglier and make no difference